CMakeLists.txt

```diff
@@ -100,7 +104,8 @@ install(FILES "${PROJECT_BINARY_DIR}/apriltag.pc" DESTINATION "${CMAKE_INSTALL_L
 
 
 # Python wrapper
-option(BUILD_PYTHON_WRAPPER "Builds Python wrapper" On)
+include(CMakeDependentOption)
+cmake_dependent_option(BUILD_PYTHON_WRAPPER "Builds Python wrapper" ON BUILD_SHARED_LIBS OFF)
```

Comment:
Can the python module not be built with static linking?

Reply:
When I tried building the wrapper with a static library it failed. I didn't look into it too much because I don't know much about it and I assumed that someone building a static library intends to statically link it into a C executable. It wouldn't surprise me if the Python wrapper required a dynamic library to work, but I don't know for sure.